boards/shields/boostxl_sensors/boostxl_sensors.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019 Brett Witherspoon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
+
+	bme280@77 {
```

Comment:
We have support for a number of sensors on this board in zephyr, please add them to the DTS.  Please also add interrupt lines for the sensors as well.

Reply:
I added the TMP007 and OPT3001 sensors to the DTS.

Our BMI160 driver only supports SPI and this board is configured for I2C. The BMM150 is connected as an auxiliary sensor to the BMI160, so it looks like I would need to initialize the BMI160 before I can use the BMM150.

The TMP007 isn't populated on newer versions of this board, so I couldn't test it. Everything looks correct though.

The OPT3001 works as expected. The driver doesn't support interrupts, but most of the newer launchpads don't have a GPIO connected to that pin anyways.

I will need to revisit #16392 so interrupts lines can be added in a compatible way.